clang/test/Sema/aarch64-sme-intrinsics/acle_sme_imm.cpp

```diff
@@ -216,9 +216,9 @@ void test_range_0_15(svbool_t pg, void *ptr) {
   SVE_ACLE_FUNC(svst1_ver_vnum_za8,,,)(0, -1, 16, pg, ptr, 1);
 
   // expected-error@+1 {{argument value 16 is outside the valid range [0, 15]}}
-  SVE_ACLE_FUNC(svldr_vnum_za,,,)(-1, 16, ptr);
+  SVE_ACLE_FUNC(svldr_vnum_za,,,)(-1, ptr, 16);
```

Comment:
Not something to fix in this patch, but we shouldn't be adding limits to the vnum parameter, or even require it to be a constant/immediate, because the SVE _vnum intrinsics also don't require this.